### ISSUE TITLE
Building Dockerfile failing after PR#613

### ISSUE THREAD

**Issue by galen-ft** [huggingface/chat-ui#637]
Hello,

Building an image from [Dockerfile](https://github.com/huggingface/chat-ui/blob/ad481ce628998761215f569c462af92b0c579040/Dockerfile#L1) is failing. The breaking change appears to have been introduced by [commit 73b316cac5a154bee40504f19e64d59eef6aa49a](https://github.com/huggingface/chat-ui/commit/73b316cac5a154bee40504f19e64d59eef6aa49a) which in turn was merged with [PR#613](https://github.com/huggingface/chat-ui/pull/613).

I use the following command to perform the build:
```bash
DOCKER_BUILDKIT=1 docker build -t $TAG -f Dockerfile .
```

It appears that there is a problem with the way `.env.local` is being read. The error message is related to `MONGODB_URL` and it is:

```bash
[...]
> [builder 3/3] RUN --mount=type=secret,id=DOTENV_LOCAL,dst=.env.local     npm run build:
#20 0.742 
#20 0.742 > chat-ui@0.6.0 build
#20 0.742 > vite build
#20 0.742 
#20 0.908 ▲ [WARNING] Cannot find base config file "./.svelte-kit/tsconfig.json" [tsconfig.json]
#20 0.908 
#20 0.908     tsconfig.json:2:12:
#20 0.908       2 │   "extends": "./.svelte-kit/tsconfig.json",
#20 0.908         ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#20 0.908 
#20 1.660 
#20 1.660 vite v4.3.9 building SSR bundle for production...
#20 1.692 transforming...
#20 4.208 "BaseClient" and "TokenSet" are imported from external module "openid-client" but never used in "src/lib/server/auth.ts".
#20 4.209 ✓ 215 modules transformed.
#20 4.292 rendering chunks...
#20 5.214 
#20 5.214 node:internal/event_target:1088
#20 5.214   process.nextTick(() => { throw err; });
#20 5.214                            ^
#20 5.216 Error: Please specify the MONGODB_URL environment variable inside .env.local. Set it to mongodb://localhost:27017 if you are running MongoDB locally, or to a MongoDB Atlas free instance for example.
```

Please note that the build succeeds on [commit e0c0b0e53fd3d9452c3adae82de39d15c9476a1b](https://github.com/huggingface/chat-ui/commit/e0c0b0e53fd3d9452c3adae82de39d15c9476a1b) which immediately precedes the [commit](https://github.com/huggingface/chat-ui/commit/73b316cac5a154bee40504f19e64d59eef6aa49a) which apparently introduced the breaking change:

![Screenshot 2023-12-15 at 11 08 36](https://github.com/huggingface/chat-ui/assets/105213101/c2cda416-2f14-4fa8-9012-c1938744de12)

Has there been a change in the way that the Chat UI docker image should be built?

Any help is much appreciated!

Fyi @nsarrazin .

**Comment by nsarrazin** [comment]
Thanks for the detailed report! I'll take a look

**Comment by galen-ft** [comment]
Okay, I've found the issue. It is on line 9 in the [.dockerignore](https://github.com/huggingface/chat-ui/blob/ad481ce628998761215f569c462af92b0c579040/.dockerignore#L9) file.

In particular, the addition of `.env*` means that the `.env.local` will not get sent to `docker build` which in turn results in the error message I've shared above.

![Screenshot 2023-12-15 at 12 43 33](https://github.com/huggingface/chat-ui/assets/105213101/08a66293-09c3-4f3a-8765-842d43b47202)

**Comment by nsarrazin** [comment]
Hey! Thanks for finding a fix for this. If you want to open a PR I can review it! To me seems like a good fix.

**Comment by galen-ft** [comment]
Thanks for the feedback, @nsarrazin! I've opened [PR#642](https://github.com/huggingface/chat-ui/pull/642).

Please let me know if I am missing something.

Suggestions are welcome.

**Comment by galen-ft** [comment]
This issue has been resolved by [PR#642](https://github.com/huggingface/chat-ui/pull/642).
